README.md

```diff
@@ -41,19 +41,31 @@ desired location.
 * Run `rmt-cli sync` to download available products and repositories data for your organization from SCC
 * Run `rmt-cli products --all` to see the list of products that are available for your organization
 * Run `rmt-cli repos --all` to see the list of all repositories available
-* Use the `rmt-cli repos enable` command to choose which product repositories to mirror, for example:
+* Use the `rmt-cli products enable` command to choose which product repositories to mirror, for example:
   ```
-  rmt-cli repos enable SLES/12.2/x86_64
+  rmt-cli products enable SLES/12.2/x86_64
   ```
   The above command would select the mandatory (`pool`, `updates`) SLES 12 SP2 repositories to be mirrored.
   Alternatively, you can specify repository ID to choose individual repositories.
 * Run `rmt-cli mirror` to mirror selected repositories and make their content available through RMT
 * Register client against RMT by running `SUSEConnect --url https://rmt_hostname`
   After successful registration the repositories from RMT will be used by `zypper` on the client machine.
 
+### Custom Repositories
+
+* Run `rmt-cli repos custom add URL NAME PRODUCT_ID` to add a new custom repository to a product, for example:
```

Comment:
`PRODUCT_ID` is no longer an allowed parameter. The step with attaching the product should be additionally documented.